python/py-nltk/Portfile

```diff
@@ -16,17 +16,24 @@ description         Natural Language Toolkit
 long_description    NLTK is Python modules for research and development in natural language processing
 homepage            http://www.nltk.org/
 
-python.versions     27 34 35 36 37 38
+checksums           rmd160  1e84cd7328d47f23bc0318a6d5eaf318006550e1 \
+                    sha256  52fef02f225b0e36cab3f5b28d66b718f25d5aee806097255a1f7991bc7c000f \
+                    size    2821586
+
+python.versions     27 37 38
 
 if {${name} ne ${subport}} {
-    checksums       rmd160  efe6688551f178b91ecefa44efb96a3544435018 \
-                    sha256  92ac34a8c6975bf7a7796012689c302297c047cecf2d4ec41e5aa77b9b8c45d0 \
-                    size    2809052
-
-    depends_build   port:py${python.version}-setuptools
-    depends_lib     port:py${python.version}-numpy \
-                    port:py${python.version}-yaml \
-                    port:py${python.version}-matplotlib
+    depends_build-append \
+                    port:py${python.version}-setuptools
+    depends_lib-append \
+                    port:py${python.version}-click \
+                    port:py${python.version}-joblib \
+                    port:py${python.version}-matplotlib \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-regex \
+                    port:py${python.version}-six \
+                    port:py${python.version}-tqdm \
+                    port:py${python.version}-yaml
```

Comment:
where do all these dependencies come from? I don't see most of them listed in upstream's ```setup.py``` for the released version (3.4.5 for which this update is). There is a largee list of dependency in the ```master``` branch, but these shouldn't be used.

Reply:
https://github.com/nltk/nltk/blob/90d6779613fc698a39229e383f93b5c69bae672b/setup.py#L102-L109

Reply:
sure, but that's upstream's ```master``` branch and not for the released version

Reply:
> sure, but that's upstream's `master` branch and not for the released version

They'll appear in the next release soon enough.

I've been adding dependencies on more bleeding edge modules undergoing rapid development and have found that it's just more efficient to keep things running to anticipate the direction and toss in the kitchen sink.

Reply:
I completely disagree on this here and in other submissions! You can add these dependencies once upstream releases a version that requires them; I will personally not merge PRs in which this happens.

Reply:
Ok they're gone.